apps/nextjs/src/components/AppComponents/Chat/chat-left-hand-side.tsx

```diff
@@ -52,10 +53,8 @@ const ChatLeftHandSide = ({
           <QuickActionButtons isEmptyScreen={!!messages.length} />
         )}
       </div>
-      <ChatPanel
-        isEmptyScreen={!!messages.length}
-        isDemoLocked={isDemoLocked}
-      />
+      <ButtonScrollToBottom />
```

Comment:
Moved out of the chat form as it was showing in an incorrect position:absolute position in the story